frontend/dynamic-demo-plugin/README.md

```diff
@@ -60,3 +60,33 @@ Following commands should be executed in Console repository root.
    ```
 
 To test a locally built demo plugin image, simply update and re-apply `oc-manifest.yaml`.
+
+
+## Enable the plugin
+
+To enable the plugin on the OpenShift cluster you need to edit console-operator's config
+There you need to list all the Console plugins that you want to enable.
+
+To update the config you can either use:
+- CLI
+
+  ```sh
+  $ oc edit console.operator.openshift.io cluster
+  ```
+
+- Console WebUI, going to
+
+  `https://<cluster-hostname>/k8s/cluster/operator.openshift.io~v1~Console/cluster/yaml`
```

Comment:
I'd just leave the CLI command for now since we can't give a link without the user needing the edit the URL.